wallet/how-to/interact-with-smart-contracts/unity/contract-interface.md

```diff
@@ -0,0 +1,150 @@
+---
+sidebar_position: 1
+---
+
+# Contract interface
+
+When [interacting with smart contracts in Unity](index.md), the `IContract` interface defines an
+interface of a given contract.
+This interface includes all the functions of the contract, and optionally includes a constructor
```

Comment:
How about this... "This interface includes all the functions of the contract, and includes an optional constructor..."?